doc/install.rst

```diff
@@ -6,6 +6,59 @@ Installing DLR
++--------------+--------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Rockchips    | RK3399       |  https://s3-us-west-2.amazonaws.com/neo-ai-dlr-release/v1.0/deeplens-x86_64-igp-ubuntu16_04-glibc2_23-libstdcpp3_4/dlr-1.0-py2.py3-none-any.whl      |
++--------------+--------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Huawei       | Hikey 960    |  https://s3-us-west-2.amazonaws.com/neo-ai-dlr-release/v1.0/hikey-aarch64-mali-linaro-4_19_5-glibc2_24-libstdcpp3_4/dlr-1.0-py2.py3-none-any.whl     |
++--------------+--------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Nvidia       | Jetson_TX1   |  https://s3-us-west-2.amazonaws.com/neo-ai-dlr-release/v1.0/jetsontx1-aarch64-cu90-ubuntu16_04-glibc2_23-libstdcpp3_4/dlr-1.0-py2.py3-none-any.whl   |
++--------------+--------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Nvidia       | Jetson_TX2   |  https://s3-us-west-2.amazonaws.com/neo-ai-dlr-release/v1.0/jetsontx2-aarch64-cu90-ubuntu16_04-glibc2_23-libstdcpp3_4/dlr-1.0-py2.py3-none-any.whl   |
++--------------+--------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Nvidia       | Jetson_Nano  |  https://s3-us-west-2.amazonaws.com/neo-ai-dlr-release/v1.0/jetsonnano-aarch64-cu10-ubuntu18_04-glibc2_27-libstdcpp3_4/dlr-1.0-py2.py3-none-any.whl  |
++--------------+--------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Raspberry    | Rasp2b       |  https://s3-us-west-2.amazonaws.com/neo-ai-dlr-release/v1.0/pi-armv7l-raspbian4.14.71-glibc2_24-libstdcpp3_4/dlr-1.0-py2.py3-none-any.whl            |
++--------------+--------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+If your device is not listed in the table, use table2. You will identify your device by the processor architecture, operating system, and versions of GLIBC and LIBSTDC++. Of note, DLR installation may depend on other configuration differences or even location of dependency libraries; if the provided wheel URL does not work, please consider compiling DLR from source (see `Building DLR from source`_ section).
+
+Table2: List of Supported Architectures (Incomplete)
```

Comment:
minor suggestion. we may want to take out the word "Incomplete". We can always add more later.